hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -333,7 +337,13 @@
         { bits: "1", name: "rxf_empty",  desc: "RX FIFO empty", resval: "1"},
         { bits: "2", name: "txf_full",   desc: "TX FIFO full"},
         { bits: "3", name: "txf_empty",  desc: "TX FIFO empty", resval: "1"},
-        { bits: "4", name: "abort_done", desc: "Abort process is completed", resval: "1" },
+        { bits: "4", name: "abort_done",
+          desc: '''Abort process is completed.
+
+            Current version does not implement abort_done logic. It is tied to
```

Comment:
do you want to implement this at D3 or just a placeholder for future enhancement?

Reply:
I think this feature is low priority. So, I prefer punting the item for the future enhancement.

Reply:
SGTM

Reply:
if we're tabling to the future, we should track with an issue :) otherwise we'll definitely forget.

Reply:
It is in #9823

Reply:
sounds good, thanks @eunchan